package.json

```diff
@@ -225,7 +226,8 @@
             ".ring": "ring",
             ".sml": "cd $dir && sml $fileName",
             ".mojo": "mojo run",
-            ".erl": "escript"
+            ".erl": "escript",
+            ".pkl": "pkl eval -f yaml $fileName -o $fileNameWithoutExt.yaml"
```

Comment:
Same here. Add `cd $dir &&`, otherwise it will fail to find the file if file is not in root of the folder.

Reply:
Applied changes